specs/valid.spec.js

```diff
@@ -64,10 +53,6 @@ describe('Valid images', function () {
         expect(asyncDimensions.width).to.be(expected.width);
         expect(asyncDimensions.height).to.be(expected.height);
 
-        if (type !== 'tiff') {
-          expect(bufferDimensions.width).to.be(expected.width);
-          expect(bufferDimensions.height).to.be(expected.height);
-        }
```

Comment:
The only thing that should be changed in this file should be the removal of `if (type !== 'tiff')`, otherwise you're removing buffer testing altogether?